apps/frontend/src/components/global/ExportCKLModal.vue

```diff
@@ -182,13 +196,15 @@
                         type="number"
                         :placeholder="profile.versionplaceholder"
                         class="pr-2"
+                        @keydown="(event) => preventNonNumeric(event)"
```

Comment:
Does not seem to handle all cases, and the behavior gets kinda weird with invalid stuff maybe turning into a 0 or something.  Better to just go back to the snackbar warning.

Reply:
Done